src/main/java/com/prgrms/vouchermanagement/voucher/repository/MemoryVoucherRepository.java

```diff
@@ -0,0 +1,39 @@
+package com.prgrms.vouchermanagement.voucher.repository;
+
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.stream.Collectors;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Repository;
+
+import com.prgrms.vouchermanagement.voucher.domain.Voucher;
+
+@Repository
+public class MemoryVoucherRepository implements VoucherRepository {
+	private final ConcurrentHashMap<UUID, Voucher> storage = new ConcurrentHashMap<>();
```

Comment:
```suggestion
	private final Map<UUID, Voucher> storage = new ConcurrentHashMap<>();
```

인터페이스를 선언하고 구현 클래스로 초기화하셔도 될 것 같네요. !